.gitlab-ci.yml

```diff
@@ -242,27 +235,13 @@ examples-contract-build:
   stage:                           examples
   <<:                              *docker-env
   script:
-    - *update-cargo-contract
+    - cargo contract -V
     - for example in examples/*/; do
         pushd $example &&
         cargo contract build &&
         popd;
       done
 
-examples-generate-metadata:
```

Comment:
why is this removed within this PR?

Reply:
We removed this command `cargo contract generate-metadata` in the upcoming v0.8.0 release, it's no found as a flag under `cargo contract build --metadata-only`.